sjb/config/test_cases/test_pull_request_jenkins_images.yml

```diff
@@ -1,6 +1,4 @@
 ---
 parent: 'test_cases/test_branch_jenkins_images.yml'
-overrides:
-  sync_repos:
-    - name: "jenkins"
-      type: "pull_request"
```

Comment:
erm you still need to sync this as a PR so you want to add s2i to this

Reply:
so does that mean https://github.com/openshift/aos-cd-jobs/blob/master/sjb/config/test_cases/test_pull_request_wildfly_images.yml is totally wrong?